downstream/modules/platform/proc-controller-ldap-security.adoc

```diff
@@ -0,0 +1,10 @@
+[id="proc_controller-ldap-security_{context}"]
+
+= controller-ldap-security
+
```

Comment:
Same as the other topic above that you created, I think the title is missing and the file name got inserted here. Hopefully, we can use Newdoc in VSCode and this will take out any need to remember to add titles to our modules. The title should be appropriate for procedure modules (starting with a gerund, for example, "Configuring LDAP security for automation controller" or something along those lines).